lib/fetch.js

```diff
@@ -42,7 +30,11 @@ export function sanitizeEventAndNews(item, type) {
     strapline: item.data[`${type}.strapline`] ?
       item.data[`${type}.strapline`].value : null,
     datetime: item.data[`${type}.timestamp`] ?
-      mapDate(item.data[`${type}.timestamp`].value) : null,
+      item.data[`${type}.timestamp`].value : null,
```

Comment:
* `lodash/fp` ;). So you can do:
```
import getOr from 'lodash/fp';
const nullifyIfNotFound = getOr(null);
...
strapline = nullifyIfNotFound(`${type}.strapline.value`, item.data)
...
```
Unfortunately, you [can't rearg curried methods](https://github.com/lodash/lodash/issues/2413) right now, otherwise it'd be cleaner:
```
strapline = nullifyIfNotFound(`${type}.strapline.value`)
```